allauth/socialaccount/providers/instagram/provider.py

```diff
@@ -23,7 +23,7 @@ def extract_extra_data(self, data):
         return data
 
     def get_default_scope(self):
-        return ["basic"]
+        return ["user_profile", "user_media"]
```

Comment:
**Issue**: By default we should request the minimum scope to get the auth flow going. If a project needs more scopes, that can always be configured. Therefore, can we drop `user_media` here as it is not needed for auth purposes?